src/project_manager/ros_package_wizard.cpp

```diff
@@ -229,7 +230,29 @@ Core::BaseFileWizard *ROSPackageWizard::create(QWidget *parent,
 {
     Q_UNUSED(parameters);
     m_wizard = new ROSPackageWizardDialog(this, parent);
-    m_wizard->setPath(parameters.defaultPath());
+
+    QString defaultPath = parameters.defaultPath();
+
+#ifdef CREATE_FOLDERS
```

Comment:
Just curious, what is intend use for this? Is there a case where we would not build it with these methods?